roles/confluent.variables/defaults/main.yml

```diff
@@ -1101,9 +1106,372 @@ control_center_health_check_user: "{{control_center_ldap_user}}"
+kafka_connect_replicator_consumer_export_certs: "{{kafka_connect_replicator_ssl_mutual_auth_enabled}}"
+kafka_connect_replicator_consumer_keytab_path: /etc/security/keytabs/kafka_connect_replicator_consumer.keytab
+kafka_connect_replicator_consumer_kerberos_keytab_path: /etc/security/keytabs/kafka_connect_replicator_consumer.keytab
+kafka_connect_replicator_consumer:
+  properties: {}
+
+### Variable to define bootstrap servers for Kafka Connect Replicator Producer.  Mandatory for Kafka Connect Replicator setup.
+kafka_connect_replicator_producer_bootstrap_servers: localhost:9092
+
+### Use to set custom Kafka Connect Replicator Producer properties. This variable is a dictionary. Put values true/false in quotation marks to perserve case.
+kafka_connect_replicator_producer_custom_properties: "{{ kafka_connect_replicator_producer.properties }}"
+
+### Boolean that determines if TLS is enabled for the Kafka Connect Replicator Producer configuration.
+kafka_connect_replicator_producer_ssl_enabled: false
+
+### Boolean that determines if mTLS is enabled for the Kafka Connect Replicator Producer configuration.
+kafka_connect_replicator_producer_ssl_mutual_auth_enabled: false
+
+### Boolean that determines if a provided keystore and truststore are being used for the Kafka Connect Replicator producer configuration.
+kafka_connect_replicator_producer_ssl_provided_keystore_and_truststore: false
+
+### Set to the location of your TLS CA Certificate when configuring TLS for Kafka Connect Replicator Producer.
+kafka_connect_replicator_producer_ssl_ca_cert_path: ""
+
+### Set to the location of your TLS signed certificate when configuring TLS for Kafka Connect Replicator Producer.
+kafka_connect_replicator_producer_ssl_cert_path: ""
+
+### Set to the location of your TLS key when configuring TLS for Kafka Connect Replicator Producer.
+kafka_connect_replicator_producer_ssl_key_path: ""
+
+### Set to the password of your TLS key when configuring TLS for Kafka Connect Replicator Producer.
+kafka_connect_replicator_producer_ssl_key_password: ""
+
+### Set to the location of your TLS TrustStore when configuring TLS using Keystores and TrustStores for Kafka Connect Replicator Producer.
+kafka_connect_replicator_producer_ssl_truststore_file_path: ""
+
+### Set to the location of your TLS KeyStore when configuring TLS using Keystores and TrustStores for Kafka Connect Replicator Producer.
+kafka_connect_replicator_producer_ssl_keystore_file_path: ""
+
+### Boolean that determines if Kerberos is enabled for the Kafka Connect Replicator Producer configuration.
+kafka_connect_replicator_producer_kerberos_enabled: "{{ true if kafka_connect_replicator_producer_kerberos_principal is defined else false }}"
+
+### Boolean that determines if SASL SCRAM is enabled for the Kafka Connect Replicator Producer configuration.
+kafka_connect_replicator_producer_sasl_scram_enabled: "{{ true if kafka_connect_replicator_producer_sasl_scram_username is defined else false }}"
```

Comment:
we've been using listener dicts elsewhere in our code this seems like a break in that pattern